src/NzbDrone.Api/Authentication/AuthenticationService.cs

```diff
@@ -74,6 +84,16 @@ public bool IsAuthenticated(NancyContext context)
                 return false;
             }
 
+            if (context.Request.IsLoginRequest())
+            {
+                return true;
+            }
+
+            if (context.Request.IsContentRequest())
+            {
+                return true;
+            }
+
             if (!Enabled)
```

Comment:
move this all the way up, just after the `IsApiRequest` check, and remove the check in `IsFeedRequest`